app/test_main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.main import can_access_google_page
+from unittest import mock
+
+
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+def test_can_access_google_page(mock_has_internet: None,
+                                mock_valid_google_url: None
+                                ) -> None:
+    mock_valid_google_url.return_value = True
+    mock_has_internet.return_value = True
+    assert can_access_google_page("http://example.com")
```

Comment:
It's better to assert, that the function result is equal to something:
for example: 
```suggestion
    assert can_access_google_page("http://example.com") == "Success"
```